docs/ingestion/index.md

```diff
@@ -0,0 +1,769 @@
+
+For an example of how to configure rollup, and of what how the feature will modify your data, check out the
+[rollup tutorial](../tutorials/tutorial-rollup.md).
+
+### Maximizing rollup ratio
+
+You can measure the rollup ratio of a datasource by comparing the number of rows in Druid with the number of ingested
+events. The higher this number, the more benefit you are gaining from rollup. One way to do this is with a
+[Druid SQL](../querying/sql.md) query like:
+
+```sql
+SELECT SUM("cnt") / COUNT(*) * 1.0 FROM datasource
+```
+
+In this query, `cnt` should refer to a "count" type metric specified at ingestion time. See
+[Counting the number of ingested events](schema-design.md#counting) on the "Schema design" page for more details about
+how counting works when rollup is enabled.
+
+Tips for maximizing rollup:
+
+- Generally, the fewer dimensions you have, and the lower the cardinality of your dimensions, the better rollup ratios
+you will achieve.
+- Use [sketches](#sketches) to avoid storing high cardinality dimensions, which harm rollup ratios.
+- Adjusting `queryGranularity` at ingestion time (for example, using `PT5M` instead of `PT1M`) increases the
+likelihood of two rows in Druid having matching timestamps, and can improve your rollup ratios.
+- It can be beneficial to load the same data into more than one Druid datasource. Some users choose to create a "full"
+datasource that has rollup disabled (or enabled, but with a minimal rollup ratio) and an "abbreviated" datasource that
+has fewer dimensions and a higher rollup ratio. When queries only involve dimensions in the "abbreviated" set, using
+that datasource leads to much faster query times. This can often be done with just a small increase in storage
+footprint, since abbreviated datasources tend to be substantially smaller.
+- If you are using a [best-effort rollup](#best-effort-rollup) ingestion configuration that does not guarantee perfect
+rollup, you can potentially improve your rollup ratio by switching to a guaranteed perfect rollup option, or by
+[reindexing](data-management.md#compaction-and-reindexing) your data in the background after initial ingestion.
+
+### Best-effort rollup
+
+Some Druid ingestion methods guarantee _perfect rollup_, meaning that input data are perfectly aggregated at ingestion
+time. Others offer _best-effort rollup_, meaming that input data might not be perfectly aggregated and thus there could
+be multiple segments holding rows with the same timestamp and dimension values.
+
+In general, ingestion methods that offer best-effort rollup do this because they are either parallelizing ingestion
+without a shuffling step (which would be required for perfect rollup), or because they are finalizing and publishing
+segments before all data for a time chunk has been received, which we call _incremental publishing_. In both of these
+cases, records may end up in different segments that are received by different, non-shuffling tasks cannot be rolled
```

Comment:
This sentence needs to be reworded. Perhaps something like: non-shuffling tasks cannot be -> non-shuffling tasks and cannot be

Reply:
I replaced this sentence:

> In both of these cases, records may end up in different segments that are received by different, non-shuffling tasks cannot be rolled up together.

With this one:

> In both of these cases, records that could theoretically be rolled up may end up in different segments.